# port_id: ID number of port to set data
# data: 8 bit data to be set
uint8 port_id
uint8 data
---
# None
